sdk/tools/gen_baseaddress.py

```diff
@@ -252,7 +262,7 @@ def gen_baseaddress(self, output_file):
             postfix = ' # should be above 0x%08x' % self.address
         elif self._reserved:
             postfix = ' # reserved'
-        output_file.write('set(baseaddress_%-30s 0x%08x)%s\n' % (name, self.address, postfix))
+        output_file.write('set(baseaddress_%-22s 0x%08x)%s\n' % (name, self.address, postfix))
```

Comment:
What is the point of this change?

Reply:
See what I wrote in the PRs description:
> Also switch back the padding to 22 instead of 30 spaces,
> like we used it in older branches until
> 0.4.10-dev-307-g https://github.com/reactos/reactos/commit/82351bc6d410f020b5589cfdda628073753f78f4 for the outputs
> as that makes the output-files a bit smaller again and is enough padding
> for even the longest string 'baseaddress_msvcrt_crt_dll_startup'.
> It is the exact same amount of padding we used earlier.

Reply:
right, so it is a change for the sake of changing it.